docs/using-easybuild.md

```diff
@@ -721,6 +725,7 @@ a matching `--try-X` command line options is available:
 * `--try-amend` to try tweaking a different easyconfig parameter
     * format: `--try-amend=<param>=<value>`
     * only supports string and list-of-strings value types
+    * example: `--try-amend=sources '["<source_url>"]'`
```

Comment:
```suggestion
    * example: `--try-amend=sources='["<source_url>"]'`
```

Reply:
the first ' should probably be directly after the first = to cover the whole argument to try-amend, that's usually the better way of doing that.